openpype/hosts/maya/plugins/publish/validate_frame_range.py

```diff
@@ -53,6 +53,21 @@ def process(self, instance):
             )
             return
 
+        # Check the include handle settings of the actual project
+        include_handles_settings = context.data["project_settings"]["maya"]["include_handles"]
+        current_task = context.data["anatomyData"]["task"]["name"]
+        include_task_handles = False
+        # Define if we had to include/exclude the handles
+        for item in include_handles_settings["per_task_type"]:
+            if current_task in item["task_type"]:
+                include_task_handles = item["include_handles"]
```

Comment:
There is no `break` or check if a task type is added multiple time in different "per_task_type", don't the correct idea looping through everything and check for issues or break at first found.

Reply:
Indeed, this kind of issue could happen. I added a break in the loop and a warning message in the settings. Should be good now.